### ISSUE TITLE
Haskell Broken

### ISSUE THREAD

**Issue by qrpnxz** [Headline/discord-compiler-bot#123]
Getting this for a simple hello program:

```
/usr/bin/ld.gold: error: cannot find -lnuma

rts/posix/OSMem.c:370:0: error:
     error: undefined reference to 'mbind'

rts/posix/OSMem.c:748:0: error:
     error: undefined reference to 'numa_num_configured_nodes'

rts/posix/OSMem.c:758:0: error:
```

Discord command:
````
;compile haskell
```haskell
main = print "hello"
```
````

**Comment by 54** [comment]
any updates on this? 😄

**Comment by Headline** [comment]
Fixed as of #126
